app/templates/search/catalogue/accord.html

```diff
@@ -0,0 +1,308 @@
+                      <div class="tna-date-input__item">
+                        <label for="date-day" class="tna-date-input__item-label">
+                          Day
+                        </label>
+                        <input type="text" id="date-day" value="" name="date-day" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-month" class="tna-date-input__item-label">
+                          Month
+                        </label>
+                        <input type="text" id="date-month" value="" name="date-month" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-year" class="tna-date-input__item-label">
+                          Year
+                        </label>
+                        <input type="text" id="date-year" value="" name="date-year" class="tna-date-input__item-input tna-date-input__item-input--wider" inputmode="numeric">
+                      </div>
+                    </div>
+                    <legend class="tna-form__legend">
+                      To:
+                    </legend>
+                    <div class="tna-date-input" id="date">
+                      <div class="tna-date-input__item">
+                        <label for="date-day" class="tna-date-input__item-label">
+                          Day
+                        </label>
+                        <input type="text" id="date-day" value="" name="date-day" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-month" class="tna-date-input__item-label">
+                          Month
+                        </label>
+                        <input type="text" id="date-month" value="" name="date-month" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-year" class="tna-date-input__item-label">
+                          Year
+                        </label>
+                        <input type="text" id="date-year" value="" name="date-year" class="tna-date-input__item-input tna-date-input__item-input--wider" inputmode="numeric">
+                      </div>
+                    </div>
+                  </fieldset>
+                </div>
+              
+              </div>
+
+              <div class="tna-container tna-container--align-right">
+                  <div class="tna-column tna-column--width-1-3">
+                    <input type="submit" value="Update" class="tna-button update-filters">
+                  </div>
+                </div>
```

Comment:
I see the intention of this is to move the button to the right but I would argue that we don't need to do that. I feel we should use the same submit button as we did before: https://github.com/nationalarchives/ds-search/blob/main/app/templates/search/catalogue/index.html#L29-L36

Reply:
I have adapted the code to use the nunjucks component